tests/test_repository.py

```diff
@@ -114,37 +56,19 @@ def f(pkg_name: str) -> bool:
 @pytest.mark.skipif(
     OS_VERSION == "tumbleweed", reason="No testing for openSUSE"
 )
-@pytest.mark.parametrize(
-    "container_per_test", [REPOCLOSURE_CONTAINER], indirect=True
-)
-def test_repoclosure(container_per_test):
-    """Run :command:`dnf repoclosure` on all packages in the ``SLE_BCI`` repository
-    excluding the packages in :py:const`REPOCLOSURE_FALSE_POSITIVES`.
-
-    """
-    package_list = list(
-        filter(
-            package_name_filter_func(
-                REPOCLOSURE_FALSE_POSITIVES + KNOWN_BROKEN
-            ),
-            get_package_list(container_per_test.connection),
-        )
+@pytest.mark.parametrize("container_per_test", [BASE_CONTAINER], indirect=True)
+def test_installcheck(container_per_test):
+    """Run installcheck against the SLE_BCI repo + locally installed packages."""
+    container_per_test.connection.run_expect(
+        [0],
+        "zypper ref && installcheck $(uname -m) --exclude 'rpm-ndb' /var/cache/zypp/solv/SLE_BCI/solv --nocheck /var/cache/zypp/solv/@System/solv",
```

Comment:
Could you add a short explanation what these two paths are and why one of them is behind `--nocheck`? I am not familiar with how installcheck works

Reply:
Done